### ISSUE TITLE
Crashing on Windows

### ISSUE THREAD

**Issue by Evanfeenstra** [glawson/electron-deeplink#25]
Hi! thanks for the great package. In production, my Windows apps crashes immediately. It seems the error is coming from this line 115 in src/index.ts

```
if (os.platform() === 'win32') {
            app.setAsDefaultProtocolClient(protocol, process.execPath, [path.resolve(process.argv[1])]);
        }
```

I guess `process.argv[1]` is `undefined`? 

Using electron-builder FYI

**Comment by glawson** [comment]
thanks @Evanfeenstra.  Please try this branch `npm install https://github.com/glawson/electron-deeplink.git#fixes-uncaught-on-windows`.  Let me know if it works, I'll release it if so.

**Comment by Evanfeenstra** [comment]
It works!! Thank u!

**Comment by glawson** [comment]
@Evanfeenstra, good deal.  I just released that fix as 1.0.6.

**Comment by glawson** [comment]
Closed to #24
